apps/website/src/content/docs/flex.mdx

```diff
@@ -1,19 +1,43 @@
 ---
 title: Flex
-description: A utility component to work with CSS flexbox and iTwinUI design tokens.
+description: A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens.
 thumbnail: #TODO
 group: utilities
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='flex--basic' />
+## Usage
+
+Wrap `Flex` around the components that should be the children of the flex container. Set the gap using the `gap` prop with a value of an iTwinUI size token (e.g. `x`, `xs`) or any valid css value (e.g. "1rem"). The default `gap` is `xs`.
 
 <LiveExample src='Flex.main.jsx'>
   <AllExamples.FlexMainExample client:load />
 </LiveExample>
 
-A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
+The `Flex` container offers props that are equivalent to those of the CSS flexbox, such as `alignItem`, `justifyContent`, `flexDirection`, and `flexWrap`.
+
+### Spacer
+
+The `Flex.Spacer` subcomponent fills up the available space between flex items. This subcomponent is useful when it is necessary to push certain items to the edges.
+
+<LiveExample src='Flex.spacer.jsx'>
+  <AllExamples.FlexSpacerExample client:load />
+</LiveExample>
+
+### Flex Items
+
+This `Flex.Item` allows customizing flex items through the `flex`, `alignSelf` and `gapBefore`/`gapAfter` props.
```

Comment:
non-blocking (can be done in a future PR): I would like to explain in a bit more detail how the `flex` prop works, because our users are often confused by why we don't offer `flexGrow` and `flexShrink` props.

Reply:
also, phrasing suggestion

```suggestion
The `Flex.Item` subcomponent allows customizing flex items through the `flex`, `alignSelf` and `gapBefore`/`gapAfter` props.
```

Reply:
I have added this as an after-PR todo and pushed the suggested change.